src/machine.js

```diff
@@ -44,8 +44,16 @@ class Machine {
   }
 
   getEdgeFromTransition(transition: TransitionObject): Edge {
+    if (!this.#state) {
+      this.log('Invalid State - unable get Edges for Transistion');
+      return false;
+    }
+    if (!transition) {
```

Comment:
this should be handled in `verifyEdge`, you should rather check that `transition` supports a `.toString()` call